idx/memory/memory_test.go

```diff
@@ -96,6 +98,7 @@ func TestGetAddKey(t *testing.T) {
 }
 
 func TestFind(t *testing.T) {
+	log.NewLogger(0, "console", fmt.Sprintf(`{"level": %d, "formatting":false}`, 1))
```

Comment:
what does this do? i don't see any log calls in any of the tests in `idx/memory` or is this to make the log calls in memory.go work?